### DISCUSSION TITLE
已经支付成功后，点击我已付款提示 您还未完成付款，请继续付款

### DISCUSSION THREAD

**Discussion by ityz-com** [yansongda/pay#617] [Q&A]
支付环境在微信里面，打开后通过支付提示跳转外部浏览器支付成功并且调用了，接受回调跟确认回调后（支付复制文档回调的方法）返回微信后返回微信里面还是提示您还未完成付款，请继续付款，请问大佬们怎么解决的

![6PJM))M_52)@1{7YBBAN1`F](https://user-images.githubusercontent.com/20999404/170630024-17a332b5-6221-434a-a9e0-215695ade88f.jpg)

**Comment by yansongda** [comment]
你先用正常的浏览器测试是不是正常比较好？这种自己排查下就行了

**Comment by ityz-com** [comment]
我把当前地址复制到浏览器里面执行也是这个提示，（该订单是已经成功了）

我接受回调方法是
`Pay::config($this->config);
// 是的，你没有看错，就是这么简单！
$result = Pay::alipay()->callback();`
